WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -31,7 +31,7 @@ public class CoinJoinTabViewModel : WalletActionViewModel
 		private CompositeDisposable Disposables { get; set; }
 
 		private long _roundId;
-		private RoundPhase _phase;
+		private PhaseState _phase;
```

Comment:
The backing field should be named the same as the property field with low-capital and _.

Reply:
done